site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -1351,8 +1351,11 @@ components:
                                  In this case the id field must be set instead.
                  "pwd-hash":
                     type: string
-                    format: byte
-                    description: The Base64 encoding of the password hash created using the `hash-function` and (optional) `salt`.
+                    description: The password hash as a string, either the output of applying the Bcrypt hash function
+                                 to the clear text password and (optional) `salt` if the `hash-function` is `bcrypt`, or otherwise
```

Comment:
the value in the *salt* field is actually only used if the hash function is *not* `bcrypt`. Bcrypt generates a random salt value as part of the hash algorithm and includes it in the produced hash.

Reply:
Thank you. I changed it.